raiden/app.py

```diff
@@ -127,7 +140,29 @@ def app(privatekey, eth_rpc_endpoint, registry_contract_address,
     config = App.default_config.copy()
     config['host'] = listen_host
     config['port'] = listen_port
-    config['privatekey_hex'] = privatekey
+
+    accmgr = AccountManager(keystore_path)
+    if not accmgr.address_in_keystore(address):
+        addresses = list(accmgr.accounts.keys())
+        formatted_addresses = ["[{:3d}] - 0x{}".format(idx, addr) for idx, addr in enumerate(addresses)]
```

Comment:
I think this is a bit too verbose, for index and address especially when it's an iterator in an one liner.